custom_components/homewhiz/cloud.py

```diff
@@ -190,7 +191,13 @@ async def send_command(self, command: Command) -> None:
     @callback
     def handle_notify(self, payload: str) -> None:
         message = from_dict(MqttPayload, json.loads(payload))
-        offset = int(message.state.reported.wfaStartOffset)
+        if message.state.reported.wfaStartOffset:
+          offset = int(message.state.reported.wfaStartOffset)
+          self._last_wfaStartOffset = offset
+        elif self._last_wfaStartOffset:
+          offset = self._last_wfaStartOffset
+        else:
+          return
```

Comment:
Can you explain to me why you added this return statement? I'm not too familiar with the cloud code (I only have a Bluetooth device), but in case message.state.reported.wfaStartOffset does not exist, and the previous offset (self._last_wfaStartOffset) is 0, the data would not be updated.

Reply:
+ Although the current code is lacking comments, I would appreciate if you could include a small comment that explains why this workaround is needed 😄 
Don't want to see this code being deleted because we forgot why it was there 😕